### PR TITLE
Action persistance with events

### PR DESCRIPTION
**Pull request by blackshadev**
Add persistent of actions through the use of an scheduled command, an event and listener.

## How to test

- After migrating (or migrate fresh, because the actions are only created when running the command)
- Run `artisan events:finalize`
- See actions for Jon Snow and Ranonkeltje

### CODE REVIEW

File: app/Services/ActionGenerator/EventStraightFlushActionApplicator.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            ->map(fn (object $data) => $data['code'])
+            ->toArray();
+
+        $codes = array_unique($codes);
+
+        if (in_array('K', $codes, true) && in_array('N', $codes, true)) {
+            $codes = array_diff($codes, ['K']);
+        }
+
+        return ! (count($codes) < 5);
```
Comment (robinklaassen) [inline app/Services/ActionGenerator/EventStraightFlushActionApplicator.php:55]: Also a constant for this `5`

**Review by robinklaassen — APPROVED** [review]
Looks good, have some small requests to help our future devs :P but nothing blocking

File: tests/Unit/Listeners/AddMemberActionForFinishedEventTest.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    {
+        /** @var Event $event */
+        $event = Event::create();
+        /** @var Member $member */
+        $member = Member::create();
+        $event->members()->save($member);
+
+        $applicator1 = \Mockery::mock(EventActionApplicator::class);
+        $applicator1
+            ->expects('shouldApply')
```
[the remark this answers is not in the document]
Comment (blackshadev) [reply tests/Unit/Listeners/AddMemberActionForFinishedEventTest.php:30]: Nope, I extended from the laravel base TestCase, which already does that for you. 
see \Illuminate\Foundation\Testing\TestCase::tearDown

**Comment by robinklaassen** [comment]
This PR closes #201 right?

**Pull request merged**